### ISSUE TITLE
SA: Get rear van door position

### ISSUE THREAD

**Issue by B1ack-Wh1te** [CookiePLMonster/SilentPatch#93]
CCarEnterExit::GetPositionToOpenCarDoor (0x64E77B + 6).

pHandling->AnimGroup is compared to ANIM_VEH_VAN (101), causing this condition to never be met. The reason is that pHandling->AnimGroup stores indexes for CVehicleAnimGroupData::m_vehicleAnimGroups. This index is calculated as follows: any animation group (for example, the same ANIM_VEH_VAN) is taken and ANIM_VEH_STD is subtracted from it. Hence, for the condition to work properly, you need a comparison with ANIM_VEH_VAN - ANIM_VEH_STD. This will be equal to 13.

**Comment by CookiePLMonster** [comment]
What gameplay consequences does this have?

**Comment by B1ack-Wh1te** [comment]
Members of the player's gang, when trying to sit on the back seats of the van, will not go to the center of the doors, but a little to the left/right of them, and then as if abruptly teleport to the center of the doors themselves. It's a little-noticed bug, but it's triggered by incorrect code, similar to that bug with headlights in foggy and rainy.

**Comment by CookiePLMonster** [comment]
I suspect III and VC may also be affected by this?

**Comment by B1ack-Wh1te** [comment]
I don't think so. There is a completely different code and there is no such field in handling.
